### DISCUSSION TITLE
3.5 inch rpi lcd on CB1

### DISCUSSION THREAD

**Discussion by DmitriyAD** [bigtreetech/CB1#52] [Q&A]
How do I start the screen 3.5 inch rpi lcd  on the BIGTREETECH PI4B Adapter V1.0 and CB1 ?

**Comment by ghostv714** [comment]
i think it only can be some kind of hdmi. the only thing is steeing up the usb. dose the cb1 have a setting gui like raspi config ? ifso what is it ?
